src/agents/simple.py

```
@@ -1,50 +1,39 @@
-            content=f"Llevamos {num_mensajes} mensajes en total. El historial se mantiene automáticamente."
-        )
-    
-    # Retornar el nuevo mensaje (se agrega automáticamente al historial)
-    return {
-        "messages": [respuesta],
-        "contador_interacciones": nuevo_contador
-    }
-
-
-# Construir el grafo
+        new_state["my_age"] = random.randint(20, 30)
```

Ruff (0.13.3)

Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)